src/tools/PrizeTierController/ProjectionView/PrizePoolItem.tsx

```diff
@@ -0,0 +1,409 @@
+import {
+  PrizeTierConfigV2,
+  PrizeTierHistoryContract,
+  ProjectionSettings
+} from '@prizeTierController/interfaces'
+import { PrizeTierHistoryTitle } from '@prizeTierController/PrizeTierHistoryTitle'
+import { calculateDprMultiplier } from '@prizeTierController/utils/calculateDprMultiplier'
+import { calculateEstimatedTimeFromPrizeChance } from '@prizeTierController/utils/calculateEstimatedTimeFromPrizeChance'
+import { formatPrettyPercentage } from '@prizeTierController/utils/formatPrettyPercentage'
+import { getTimeBasedDrawValue } from '@prizeTierController/utils/formatTimeBasedDrawValue'
+import classNames from 'classnames'
+import { formatUnits } from 'ethers/lib/utils'
+import { useAtom } from 'jotai'
+import { useTranslation } from 'next-i18next'
+import { useEffect, useMemo } from 'react'
+import {
+  FieldErrorsImpl,
+  useForm,
+  UseFormRegister,
+  UseFormSetValue,
+  UseFormWatch
+} from 'react-hook-form'
+
+export const PrizePoolItem = (props: {
+  prizePool: PrizePool
+  prizeTierHistoryContract: PrizeTierHistoryContract
+}) => {
+  const { prizePool, prizeTierHistoryContract } = props
+  const [combinedPrizeTiers] = useAtom(allCombinedPrizeTiersAtom)
+  const { data: upcomingPrizeTier, isFetched } = usePrizeTierHistoryData(prizeTierHistoryContract)
+  const { t } = useTranslation()
+
+  const prizeTier = useMemo(() => {
+    if (!!prizePool && !!prizeTierHistoryContract && !!combinedPrizeTiers) {
+      const editedPrizeTier =
+        combinedPrizeTiers[prizePool.chainId]?.[prizeTierHistoryContract.address]
+      if (!!editedPrizeTier) {
+        return editedPrizeTier
+      } else if (isFetched) {
+        return upcomingPrizeTier
+      }
+    }
+  }, [prizePool, prizeTierHistoryContract, combinedPrizeTiers, isFetched])
+
+  return (
+    <li className='p-4 bg-actually-black bg-opacity-10 rounded-xl'>
+      <PrizeTierHistoryTitle prizeTierHistoryContract={prizeTierHistoryContract} className='mb-4' />
+      {!!prizeTier ? (
+        <PrizePoolProjections
+          prizePool={prizePool}
+          prizeTierHistoryContract={prizeTierHistoryContract}
+          prizeTier={prizeTier}
+          defaultFormValues={{ tvl: '0', variance: '0' }}
+        />
+      ) : (
+        t('loading')
+      )}
+    </li>
+  )
+}
+
+const PrizePoolProjections = (props: {
+  prizePool: PrizePool
+  prizeTierHistoryContract: PrizeTierHistoryContract
+  prizeTier: PrizeTierConfigV2
+  defaultFormValues: ProjectionSettings
+}) => {
+  const { prizePool, prizeTierHistoryContract, prizeTier, defaultFormValues } = props
+  const { data: tvl, isFetched: isFetchedTvl } = usePrizePoolTvl(prizePool)
+  const {
+    register,
+    watch,
+    setValue,
+    formState: { errors }
+  } = useForm<ProjectionSettings>({
+    mode: 'onChange',
+    defaultValues: defaultFormValues,
+    shouldUnregister: true
+  })
+  const { t } = useTranslation()
+
+  const formTvl = watch('tvl')
```

Comment:
nit: Always better to put this keys into a constant somewhere. If we ever need to make changes doing it in one spot is way easier